src/Engine/Graphics/OpenGL/RenderOpenGL.cpp

```diff
@@ -195,37 +196,27 @@ RenderOpenGL::~RenderOpenGL() { logger->info("RenderGl - Destructor"); }
 
 void RenderOpenGL::Release() { logger->info("RenderGL - Release"); }
 
-Color *RenderOpenGL::ReadScreenPixels() {
-    Color *sPixels = new Color[outputRender.w * outputRender.h];
+RgbaImage RenderOpenGL::ReadScreenPixels() {
+    RgbaImage result = RgbaImage::uninitialized(outputRender.w, outputRender.h);
```

Comment:
I believe `dirty` takes more brain cycles to process for the uninitiated then just spelling out explicitly that you want uninitialized memory.

Reply:
it's not when you're used to it and it's also helpful for people who work on the graphics
anyways that's not a blocker so i'll just leave it here in case other people want to weight it